project-base/src/Shopsys/ShopBundle/Resources/definition-with-filter/product/1.json

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
What are these JSONs for?

Reply:
It's a index setting for Elasticsearch – defines what data and how are stored in Elasticsearch.
More can be found in https://github.com/shopsys/shopsys/blob/master/docs/model/product-search-via-elasticsearch.md#elasticsearch-index-setting